### PR TITLE
Introduce GetUUID API

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: comid/instance.go [hunk 1]
```diff
@@ -65,6 +65,15 @@ func (o Instance) GetUEID() (eat.UEID, error) {
 	}
 }
 
+func (o Instance) GetUUID() (string, error) {
```
[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply comid/instance.go:68]: I aligned it with the usage. I suppose UUID is used as a string by the user, then more comfortable for them to expect a string ?

[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply comid/instance.go:68]: Better align with UEID so have now corrected it!

File: comid/instance_test.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInstance_GetUUID_OK(t *testing.T) {
+	inst := NewInstanceUUID(uuid.UUID(TestUUID))
+	assert.NotNil(t, inst)
```
[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply comid/instance_test.go:12]: Done!

**Review by thomas-fossati — APPROVED** [review]
LGTM

**Pull request merged**